packages/aws-serverless/src/sdk.ts

```diff
@@ -333,7 +333,7 @@ export function wrapHandler<TEvent, TResult>(
     // Only start a trace and root span if the handler is not already wrapped by Otel instrumentation
     // Otherwise, we create two root spans (one from otel, one from our wrapper).
     // If Otel instrumentation didn't work or was filtered by users, we still want to trace the handler.
-    // TODO(v9): Since bumping the OTEL Instrumentation, this is likely not needed anymore, we can possibly remove this
+    // TODO(v9): Since bumping the OTEL Instrumentation, this is likely not needed anymore, we can possibly remove this (can be done whenever since it would be non-breaking)
```

Comment:
should we just remove the (v9) suffix here, possibly? 🤔

Reply:
yeah I guess - I originally wanted to keep it because it's a somewhat valuable priority gauge when grepping for `TODO(`